### PR TITLE
UCL-341 Fix input & text area to align with design

### PR DESCRIPTION
**Pull request by pawel-napieracz**

### CODE REVIEW

File: src/components/_BaseStyling_/BaseStyling.tsx [hunk 1]
```diff
@@ -210,9 +214,13 @@ export const BaseStyling = ({ children, properties = {} }: Props) => {
     buttonOutlineHoverTextColor: "var(--color-primary600)",
     inputBorderRadius: "2px",
     inputBorderWidth: "1px",
-    inputBorderWidthFocus: "2px",
+    inputBorderWidthFocus: "1px",
     inputBorderStyle: "solid",
+    inputBorderColor: "var(--color-blue-grey500)",
     inputBackgroundColor: "var(--light)",
+    inputHelperTextColor: "var(--color-blue-grey500)",
+    inputHoverBackgroundColor: "var(--color-blue-grey25)",
```
Comment (MichalGorskiOnegini) [inline src/components/_BaseStyling_/BaseStyling.tsx:222]: this is correct, but you actually use wrong background

File: src/components/Form/Input/Input.module.scss [hunk 2]
```diff
@@ -110,6 +115,7 @@
 
   &:disabled {
     cursor: not-allowed;
+    color: var(--greyed-out);
```
[the remark this answers is not in the document]
Comment (MichalGorskiOnegini) [reply src/components/Form/Input/Input.module.scss:118]: Just found out that there is a ticket on that created by @jashaj https://onewelcome.atlassian.net/browse/UCL-326

[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Form/Input/Input.module.scss:118]: it's a color of the text. And it should be bluegrey/400 according to design:

File: src/components/Form/Textarea/Textarea.module.scss [hunk 3]
```diff
@@ -42,21 +51,27 @@
...
+  background-color: transparent;
+  z-index: 1;
+  /* making space for outline in text area when scroll bar is visible: width smaller and margin top & bottom to have space for border */
+  width: calc(100% - var(--input-border-width));
+  margin: var(--input-border-width) 0;
 
   &:disabled {
-    background-color: var(--disabled);
     cursor: not-allowed;
+    color: var(--greyed-out);
```
[the remark this answers is not in the document]
Comment (pawel-napieracz) [reply src/components/Form/Textarea/Textarea.module.scss:74]: same as on design ;)

File: src/components/_BaseStyling_/BaseStyling.tsx (hunk 1 above)
Comment (pawel-napieracz) [reply src/components/_BaseStyling_/BaseStyling.tsx:222]: it's not ;)
Screenshot 2023-07-20 at 07 57 29
Screenshot 2023-07-20 at 07 58 02

**Review by MichalGorskiOnegini — APPROVED** [review]

**Pull request merged**